packages/react-native-otlp/test-project/android/app/build.gradle.kts

```diff
@@ -0,0 +1,77 @@
+plugins {
+    alias(libs.plugins.android.application)
+    alias(libs.plugins.kotlin.android)
+
+    id("io.gitlab.arturbosch.detekt") version("1.23.6")
+}
+
+android {
+    namespace = "io.embrace.rnembraceotlptest"
+    compileSdk = 34
+
+    defaultConfig {
+        applicationId = "io.embrace.rnembraceotlptest"
+        minSdk = 24
+        targetSdk = 34
+        versionCode = 1
+        versionName = "1.0"
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+            isMinifyEnabled = false
+            proguardFiles(
+                getDefaultProguardFile("proguard-android-optimize.txt"),
+                "proguard-rules.pro"
+            )
+        }
+    }
+
+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_17.toString()
+    }
+
+    useLibrary("android.test.mock")
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_17
+        targetCompatibility = JavaVersion.VERSION_17
+    }
+
+    buildToolsVersion = "33.0.0"
+    ndkVersion = "26.1.10909125"
+}
+
+dependencies {
+    implementation(libs.androidx.core.ktx)
+    implementation(libs.androidx.appcompat)
+    implementation(libs.material)
+
+    // react native 0.75.1
+    implementation("com.facebook.react:react-android:0.75.1")
+
+    // using package locally
+    implementation(project(":react-native-otlp"))
```

Comment:
@jpmunz please, have a look at this config. this is how I'm connecting the module into the `test-project` without copying/pasting the source files

Reply:
yep this is great, much better approach